ansible/roles/operator-pipeline/templates/openshift/tasks/preflight.yml

```diff
@@ -97,5 +97,5 @@ spec:
         preflight check operator $(params.bundle_image)
 
         echo -n $PFLT_LOGFILE | tee $(results.log_output_file.path)
-        echo -n results.json | tee $(results.result_output_file.path)
+        echo -n $PFLT_ARTIFACTS/results.json | tee $(results.result_output_file.path)
```

Comment:
Is the path to result configurable? The line below already defines a path to the artifact directory and uploads everything within the dir. I would rather prefer to store results outside an artifacts dir because otherwise we will end up with a duplicated test result in our database.

Reply:
@Allda 

It's not currently, but Preflight can make it so. I'll work on implementing.

The alternative to that would have been issuing a `mv $PFLT_ARTIFACTS/results.json <wherever-you-want-it>`. I'm thinking this is a bit more of a workaround than what we'd want to implement here.

To that end, would something like a `PFLT_RESULTFILE=/path/to/result.json` be acceptable? In this case, we'll still default to writing the results file to `$PFLT_ARTIFACTS/results.json` if that' unset. 

WDYT?

Reply:
I went ahead and updated the PR to mv the artifacts into the place the script expects for the time being as that path should work today out of the box.